its/src/test/java/com/sonar/it/csharp/ProjectLevelDuplicationTest.java

```diff
@@ -52,7 +52,7 @@ public void containsOnlyOneProjectLevelIssue() throws Exception {
 
     List<Issues.Issue> issues = getIssues("ProjectLevelDuplication")
```

Comment:
`projectLevelIssues` might be a better name

Reply:
Renamed it.